string robot_state
string delivery_check
string joy_state
float64 tray 

